packages/page-accounts/src/modals/ProxyOverview.tsx

```diff
@@ -147,22 +147,22 @@ function NewProxy ({ index, onChangeAccount, onChangeType, onRemove, proxiedAcco
   );
 }
 
-function getProxyTypeInstance (api: ApiPromise, index = 0): NodeRuntimeProxyType {
+function getProxyTypeInstance(api: ApiPromise, index = 0): NodeRuntimeProxyType {
   // This is not perfect, but should work for `{Kusama, Node, Polkadot}RuntimeProxyType`
   const proxyNames = api.registry.lookup.names.filter((name) => name.endsWith('RuntimeProxyType'));
 
   // fallback to previous version (may be Substrate default), when not found
   return api.createType<NodeRuntimeProxyType>(proxyNames.length ? proxyNames[0] : 'ProxyType', index);
 }
 
-function getProxyOptions (api: ApiPromise): { text: string; value: number; }[] {
+function getProxyOptions(api: ApiPromise): { text: string; value: number; }[] {
   return getProxyTypeInstance(api).defKeys
     .map((text, value) => ({ text, value }))
     // Filter the empty entries added in v14
     .filter(({ text }) => !text.startsWith('__Unused'));
 }
 
-function ProxyOverview ({ className, onClose, previousProxy: [existing] = EMPTY_EXISTING, proxiedAccount }: Props): React.ReactElement<Props> {
+function ProxyOverview({ className, onClose, previousProxy: [existing] = EMPTY_EXISTING, proxiedAccount }: Props): React.ReactElement<Props> {
```

Comment:
```suggestion
function ProxyOverview ({ className, onClose, previousProxy: [existing] = EMPTY_EXISTING, proxiedAccount }: Props): React.ReactElement<Props> {
```

Fix, CI fails linting due to this.